exploration/number-selection.md

```diff
@@ -460,3 +533,21 @@ and they _might_ converge on some overlap that users could safely use across pla
 #### Cons
 
 - No guarantees about interoperability for a relatively core feature.
+
+## Alternatives Considered (`key` matching)
+
+### Standardize the Serialization Forms
+
+Using the design above, remove the integer-only and no-sig-digits restrictions from LDML45
+and specify numeric matching by specifying the form of matching `key` values.
+Comparison is as-if by string comparison of the serialized forms, just as in LDML45.
```

Comment:
Given that this alternative leaves "specifying the form of matching `key` values" as undefined, I can't tell what selecting this alternative would mean. This should be either dropped, or defined more precisely.

Reply:
I agree that I didn't flesh this out fully, depending instead on the example that we were discussing [just above](https://github.com/unicode-org/message-format-wg/pull/859#discussion_r1717349732). I will add the details here.

Note well: I'm not married to this as the design, but I'm trying to get at the technical requirements, especially the expectations of message authors (including translators)

Reply:
This still needs resolution.